x-pack/test/functional_endpoint/page_objects/endpoint_alerts_page.ts

```diff
@@ -5,16 +5,40 @@
  */
 
 import { FtrProviderContext } from '../ftr_provider_context';
+import { WebElementWrapper } from '../../../../test/functional/services/lib/web_element_wrapper';
+// import { contains } from 'vega-lite/build/src/util';
 
 export function EndpointAlertsPageProvider({ getService }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
+  const parseStyle = styles =>
```

Comment:
this is parseStyle as opposed to parseStyles (with the 's') - its hard to follow if the usage is right here.

Reply:
One it is parseStyle  from string to each style
the parseStyles combines all the styles in to the Array of styles